server/data/educationAndWorkPlanClient.ts

```diff
@@ -60,10 +61,21 @@ export default class EducationAndWorkPlanClient {
     prisonNumber: string,
     updateInductionRequest: UpdateInductionRequest,
     token: string,
-  ): Promise<void> {
+  ): Promise<never> {
```

Comment:
I'm led to believe that `never` is better than `void` when there is no return value in the resolved promise 🤷‍♂️